test_psa_target.py

```diff
@@ -177,69 +189,109 @@ def _erase_flash_storage(args, suite):
         )
         sys.exit(1)
 
+    return binary_name
 
-def _execute_test(args, suite):
+
+def _execute_test():
     """
-    Execute test by using the mbed host test runner
-    :param args: Command-line arguments
-    :param suite: Test suite
+    Execute greentea runs test as specified in test_spec.json
     """
-    logging.info("Executing tests for - %s suite..." % suite)
-
-    _erase_flash_storage(args, suite)
-
-    mbed_os_dir = join(ROOT, "BUILD", args.mcu, TC_DICT.get(args.toolchain))
-    log_dir = join(ROOT, "test", "logs", args.mcu, (suite + ".log"))
-
     cmd = [
-        "mbedhtrun",
+        "mbedgt",
         "--sync=0",
-        "-p",
-        args.port,
-        "--compare-log",
-        log_dir,
         "--polling-timeout",
-        "300",
-        "-d",
-        args.disk,
-        "-f",
-        "mbed-os-tf-m-regression-tests-reset-flash.hex",
-        "--skip-reset",
-        "-C",
-        "1",
```

Comment:
Are "--skip-reset" "-C 1" required and if so, how do we pass them to gt?

Reply:
I don't think they are needed. 
-C: This either set in target.json for example:https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L5570 or there is a a default of 4 seconds.

--skip-reset - I don't know why it's done. My understanding is, currently greentea doesn't pass this to htrun. May be @jamesbeyond or tools team know @jainvikas8 knows how to pass the parameter to htrun if needed.

Reply:
It seems that we should be able to omit --skip-reset for now